### ISSUE TITLE
Build all eBPF bytecode images with multi arch support

### ISSUE THREAD

**Issue by astoycos** [bpfman/bpfman#1093]
This should already be handled with just el and eb bytecode variants let's take inspiration from https://github.com/netobserv/netobserv-ebpf-agent/blob/main/pkg/ebpf/tracer.go#L31C57-L31C90

**Comment by astoycos** [comment]
We need to follow the arch/platform bindings laid out [here](https://github.com/cilium/ebpf/blob/main/cmd/bpf2go/main.go#L47)

```
// Targets understood by bpf2go.
//
// Targets without a Linux string can't be used directly and are only included
// for the generic bpf, bpfel, bpfeb targets.
//
// See https://go.dev/doc/install/source#environment for valid GOARCHes when
// GOOS=linux.
var targetByGoArch = map[goarch]target{
	"386":      {"bpfel", "x86"},
	"amd64":    {"bpfel", "x86"},
	"arm":      {"bpfel", "arm"},
	"arm64":    {"bpfel", "arm64"},
	"loong64":  {"bpfel", "loongarch"},
	"mips":     {"bpfeb", "mips"},
	"mipsle":   {"bpfel", ""},
	"mips64":   {"bpfeb", ""},
	"mips64le": {"bpfel", ""},
	"ppc64":    {"bpfeb", "powerpc"},
	"ppc64le":  {"bpfel", "powerpc"},
	"riscv64":  {"bpfel", "riscv"},
	"s390x":    {"bpfeb", "s390"},
}
```

**Comment by Billy99** [comment]
Addressed by #1141
